crypto/crypto.ts

```diff
@@ -108,13 +108,21 @@
  */
 
 import {
-  type DigestAlgorithm as WasmDigestAlgorithm,
-  digestAlgorithms as wasmDigestAlgorithms,
+  DIGEST_ALGORITHM_NAMES,
+  type DigestAlgorithmName,
   instantiateWasm,
 } from "./_wasm/mod.ts";
-import { fnv } from "./_fnv/mod.ts";
 
-export { type WasmDigestAlgorithm, wasmDigestAlgorithms };
+export { DIGEST_ALGORITHM_NAMES, type DigestAlgorithmName };
```

Comment:
While I agree with the change, this is a breaking change. We should first deprecate `type WasmDigestAlgorithm` and `wasmDigestAlgorithms`.